src/web-components/vf-score.js

```diff
@@ -0,0 +1,137 @@
+// ## Description
+// 
+// This file implements `vf-score`, the web component that acts as the 
+// container for the entire component. `vf-score`'s shadow root holds
+// the HTML element that the renderer renders too.
+// 
+// All actual drawing is called from `vf-score`. 
+
+import Vex from '../index';
+
+export class VFScore extends HTMLElement {
+  constructor() {
+    super();
+
+    // Defaults
+    this.x = 10; // The starting x position of a system within the score.
+    this.y = 0; // The starting y position of system within the score. 
+    this.width = 500; // The entire width of the element holding the music notation. 
+    this.height = 150; // The entire height of the element holding the music notation.
+    this.stavesPerLine = 1; 
+
+    this.systemsAdded = 0; // Counter that keeps track of how many systems have dispatched 
```

Comment:
If these properties are intended to be private or protected I would document them with a jsdoc `@private` and name them with an underscore prefix so they don't seem public in IDE code-completion, API docs, etc.